test/dataloader2/test_dataloader2.py

```diff
@@ -99,6 +114,21 @@ def test_dataloader2_shutdown(self) -> None:
         data_loader: DataLoader2 = DataLoader2(datapipe=test_data_pipe)
         data_loader.shutdown()
 
+    def test_worker_exception_raised(self):
+        dp = IterableWrapper(range(100)).sharding_filter()
+        dp = MakeMistakeDataPipe(dp)
+        for worker_prefetch_cnt in [0, 5, 10]:
+            for num_workers in [1, 4]:
+                rs = PrototypeMultiProcessingReadingService(
```

Comment:
Let's use `MultiProcessingReadingService`

Reply:
Done, updated!

Reply:
Great. Can you import this PR to Phab then land it? There should be a button to do that.
![image](https://user-images.githubusercontent.com/68879799/219133222-de66a3c5-f261-4308-b6a7-ad62cc3205f9.png)